src/lib/browser.ts

```diff
@@ -145,6 +153,24 @@ export class Browser extends EventEmitter {
         this.emit('down', service)
     }
 
+    private filterService(service: Service) {
+        if(this.txtQuery === undefined) return
```

Comment:
This method doesn't really interact with the class instance . 
Why not create a utility function outside the class scope to avoid poluting the prototype , and add a additional parameter where we will pass this.txtQuery ?